install-devenv.sh

```diff
@@ -36,13 +42,17 @@ curl -fsSL \
 
 echo "${sha256}  ${tmpd}/${archive}" | /usr/bin/shasum -a 256 --check --status
 
+# Question: What is the --strip-components doing here?
```

Comment:
it conveniently removes a top level directory that is present in the archive so i don't have to do an extra `mv`